pages/community/gsoc.js

```diff
@@ -0,0 +1,100 @@
+import GenericLayout from '../../components/layout/GenericLayout'
```

Comment:
We're not participating in GSoC this year. Do we need this? 🤔

Reply:
Yes @fmvilas, it's not a 2022 page. It's a generic page for our org that contains all needed info from OUR END on what we do/how we work in GSoC. This page should also include previous years and detail when we were accepted/declined.

You should look up how most OSS orgs make a page specific for this to be used for all future years.